src/components/Header.tsx

```diff
@@ -1,22 +1,43 @@
-import React from 'react';
-import { StyleSheet, Text } from 'react-native';
+import React, { ReactElement } from 'react';
+import { StyleSheet, Text, TouchableOpacity } from 'react-native';
 
 import { GradientView } from './GradientView';
 import { getStatusBarHeight, palette, typography } from 'styles';
 
-export const Header = ({ title }: { title: string }) => {
-  return (
-    <GradientView variant={GradientView.Variant.Primary} style={styles.container}>
+interface Props {
+  title: string;
+  leftItem?: ReactElement;
+  onLeftItemPress?: () => void;
+}
+
+export const Header = ({ title, leftItem, onLeftItemPress }: Props) => (
+  <GradientView variant={GradientView.Variant.Primary} style={styles.container}>
+    <>
+      {leftItem && (
+        <TouchableOpacity style={styles.leftItemContainer} onPress={onLeftItemPress}>
+          {leftItem}
+        </TouchableOpacity>
+      )}
```

Comment:
See that on mockups we have an only back arrow or cancel button and both do the same thing. Wouldn't it be better to have it in the Header component and 'activate' it with boolean props like `isBackArrow` or `isCancelButton` ? How do you think?

Reply:
Good idea, updated :+1: